src/components/clusterprovider/azure/azure.ts

```diff
@@ -154,7 +154,7 @@ async function listLocations(context: Context): Promise<Errorable<Locations>> {
     const sr = await context.shell.exec(`az account list-locations --query ${query} -ojson`);
 
     return fromShellJson<Locations>(sr, (response) => {
-        let locations: any = {};
+        const locations: any = {};
```

Comment:
This is a scenario where I find the formal syntax confusing.  Which the `locations` _reference_ is constant, the object itself is going to be mutated.  Using `const` makes me think the object is constant, which it isn't.  I know from a language spec point of view the usage here is correct but I personally prefer `let` in these cases because it warns me to look for how the object changes over the course of its scope.  #lostbattles

Reply:
sounds good to me. fixed these!